### ISSUE TITLE
SMS Sender Enhancements - Database Updates for Provider and Description

### ISSUE THREAD

**Issue by mjones-oddball** [department-of-veterans-affairs/notification-api#1685]
## User Story - Business Need
We'd like to add new fields to assign a sender to the applicable provider and a description field so we know what use case the sender is used for.

- [ ] Ticket is understood, and QA has been contacted (if the ticket has a QA label).

### User Story(ies)

**As a** VA notify team member
**I want** to track the associated provider and use case details with the sms sender
**So that** I can support client requests and make updates over time

## Engineering Checklist

- [ ] Add new fields to sms_sender
  - [ ] Provider_id - ties back to id on providers table
    - [ ] New foreign key on the provider table
  - [ ] Description - string, meant to provide use case context since these are registered use cases
- [ ] Update models.py
- [ ] DB migration
  - [ ] Ensure downgrade works

## Acceptance Criteria

- [ ] We're able to assign the applicable provider by ID to a specific sms sender in the database
- [ ] We're able document a use case description on a specific sms sender in the database

**Comment by mjones-oddball** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/VA-Notify-620d21369d810a00146ed9c8/issues/department-of-veterans-affairs/notification-api/1685?planning-poker) with Zenhub @cris-oddball @EvanParish @k-macmillan @kalbfled @ldraney @MackHalliday @mchlwellman

**Comment by kalbfled** [comment]
I'm encountering difficulties while trying to create a migration.

**Comment by kalbfled** [comment]
@mjones-oddball @cris-oddball The acceptance criteria for this ticket doesn't make sense.  Modifying endpoints is explicitly out of scope, but that is necessary to be about to add a description or provider ID other than directly in the database.

**Comment by cris-oddball** [comment]
@kalbfled probably because that work will be done on #1687 and why this ticket has no QA label.

**Comment by mjones-oddball** [comment]
@kalbfled Correct, this ticket was purely for the database updates needed to support the new fields. That's why the AC says "In the service_sms_senders table in the database we can..." because we are not doing those actions thru postman yet. DB stuff usually happens before endpoint updates since it's a dependency. Let me know if I'm missing something? CC @k-macmillan

**Comment by kalbfled** [comment]
https://docs.sqlalchemy.org/en/20/orm/queryguide/relationships.html#summary-of-relationship-loading-styles

Lazy loading of relationships is default.
